### PR TITLE
Added checks for order type window support.

### PR DESCRIPTION
**Pull request by akallabeth**
Checking the support status for `ORDER_TYPE_WINDOW`

### CODE REVIEW

File: libfreerdp/core/window.c [hunk 1]
```diff
@@ -400,6 +400,34 @@ void update_read_window_delete_order(wStream* s, WINDOW_ORDER_INFO* orderInfo)
 	/* window deletion event */
 }
 
+static BOOL window_order_supported(const rdpSettings* settings, UINT32 fieldFlags)
+{
+	const UINT32 mask = (WINDOW_ORDER_FIELD_CLIENT_AREA_SIZE || WINDOW_ORDER_FIELD_RP_CONTENT ||
```
Comment (mfleisz) [inline libfreerdp/core/window.c]: Should be ```|``` instead of ```||```

Comment (akallabeth) [reply libfreerdp/core/window.c]: oops

**Comment by akallabeth** [comment]
@hardening @bmiklautz @nfedera @MartinHaimberger Anyone time to review and merge?

**Comment by bmiklautz** [comment]
@akallabeth would you rebase to the latest master please?

**Pull request merged**